docs/source/reference/kubernetes/index.rst

```diff
@@ -123,17 +123,35 @@ Once your cluster administrator has :ref:`setup a Kubernetes cluster <kubernetes
     $ # Set a specific namespace to be used in the current-context
     $ kubectl config set-context --current --namespace=mynamespace
 
+
+Using Custom Images
+-------------------
+By default, we use and maintain a SkyPilot container image that has conda and a few other basic tools installed.
+
+To use your own image, add :code:`image_id: docker:<your image tag>` to the :code:`resources` section of your task YAML.
```

Comment:
We should mention private registries (e.g., X, Y, Z) are supported and a pointer to  ECR secret refresh?